packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -168,6 +168,35 @@ export class RelationshipsController extends TransportController {
             if (result.error.code === "error.platform.validation.relationship.peerIsToBeDeleted") {
                 throw CoreErrors.relationships.activeIdentityDeletionProcessOfOwnerOfRelationshipTemplate();
             }
+
+            if (result.error.code === "error.platform.validation.relationshipRequest.relationshipToTargetAlreadyExists") {
+                const queryForExistingRelationship = {
+                    "peer.address": template.cache.createdBy,
+                    $or: [
+                        {
+                            ["status"]: { $eq: RelationshipStatus.Pending }
+                        },
+                        {
+                            ["status"]: { $eq: RelationshipStatus.Active }
+                        },
+                        {
+                            ["status"]: { $eq: RelationshipStatus.Terminated }
+                        },
+                        {
+                            ["status"]: { $eq: RelationshipStatus.DeletionProposed }
+                        }
+                    ]
```

Comment:
There is an operator for this:
```
                    "status": { "$in": [RelationshipStatus.Pending, RelationshipStatus.Active, ...] }
```

Reply:
Nice, that's more compact 😎 I will refactor this here and in other open PRs of mine.